mifosng-android/src/main/java/com/mifos/mifosxdroid/online/groupdetails/GroupDetailsFragment.java

```diff
@@ -325,7 +339,6 @@ public boolean onOptionsItemSelected(MenuItem item) {
             case R.id.group_notes:
                 loadNotes();
                 break;
-
```

Comment:
@TypicalGitHubUser Why you have deleted this line?

Reply:
Accidentally deleted it.